enterprise/frontend/src/embedding-sdk/components/private/SdkLicenseProblemBanner/SdkLicenseProblemBanner.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import type { ComponentStory } from "@storybook/react";
+
+import { Box } from "metabase/ui";
+
+import { SdkLicenseProblemBanner } from "./SdkLicenseProblemBanner";
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: "EmbeddingSDK/SdkLicenseProblemBanner",
```

Comment:
We don't have Loki tests configured for SDK yet. We can use `yarn storybook-embedding-sdk` in the meantime to make changes to this component easier.

![CleanShot 2567-09-11 at 05 33 35@2x](https://github.com/user-attachments/assets/4e9f2069-1074-4b77-a0d5-4bea4a291bb1)